### PR TITLE
Adds linting of openapi specs

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please include a Pivotal story at the beginning of the PR title (see below).</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Please include a description of your PR changes.</td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d67b394d99565dc3bc7bc393e7183d0b182c0da0
</p>

File: openapi/index.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
 paths:
+  "/dummy":
+    get:
+      responses:
+        "200":
+          description: "ok"
 definitions:
-  $ref: definitions.yaml
\ No newline at end of file
+  $ref: definitions.yaml
```
Comment (ioggstream) [inline openapi/index.yaml]: This is not OAS/Swagger valid. `$ref` can only be used to reference a single schema and not a full remote file.

I'm checking the `oval` package you're using.

File: openapi/index.yaml [hunk 2]
```diff
@@ -1,6 +1,12 @@
 swagger: "2.0"
 info:
+  title: "IO APIs, shared specs"
   version: "0.1"
 paths:
+  "/dummy":
```
Comment (ioggstream) [inline openapi/index.yaml]: Do we need this endpoint or a simple `{}` could suffice? 

```
paths: {}
```

Comment (cloudify) [reply openapi/index.yaml]: haven't tried, it probably works, not sure whether `oval` will complain or not

**Comment by ioggstream** [comment]
I tried and oval validated it on docker node:10 image

**Pull request merged**